### DISCUSSION TITLE
Misuratore DIN connesso Bticino Living Now e HA

### DISCUSSION THREAD

**Discussion by Salsalove** [indomus/forum#1307] [HUB: Home Assistant]
Ho cercato in giro na nessuno pare abbia parlato del misuratore DIN connesso.
So che integrando il sistema Bticino Living Now direttamente ad HA, tramite l'integrazione Homekit, non vengono esposte le entità che visualizzano i consumi dei singoli deviatori/prese connessi. Mi chiedevo invece se il misuratore DIN, che rileva l'assorbimento dell'intera abitazione, viene riconosciuto da HA oppure no. Qualcuno ci ha provato?

In caso di risposta negativa, e non volendo utilizzare nessun tipo di pennetta, che prodotto mi consigliate di utilizzare per misurare l'assorbimento e che sia integrabile in HA?

**Comment by xyzmauro** [comment]
Io uso il misuratore din ma collegato ad ha direttamente con chiavetta zigbee e Z2M senza il sistema Bticino,a naso se tramite HomeKit non vedi i consumi delle prese non credo vedrai neanche qst…di alternativa a monte dell’impianto potresti valutare Shelly EM che è wifi

**Reply by Salsalove** [reply]
Grazie ;)

**Reply by Stomodine** [reply]
confermo...mentre i comandi tapparelle puoi averli sia in app home control che in HA (tramite homekit controller) i consumi puoi leggerli O da home control O da HA con zigbee

**Reply by esposimo** [reply]
Scusate ragazzi, mi intrometto nella discussione. Quindi se in una casa si montano prese/interruttori/tapparelle della Bticino Living Now (che funzionano tutte in zigbee) + il misuratore DIN sul quadro elettrico, con un solo DIN e la chiavetta zigbee, posso leggere tutto quanto ? Io ho sempre pensato fosse necessario usare degli shelly (uno per presa, se si considera lo shelly 1PM)

**Reply by Stomodine** [reply]
Esatto..però stai attento: come ho detto sopra le tapparelle puoi integrarle con homekit controller e anche con app ticino mentre la lettura/controllo delle prese o da HA o da app ticino
